src/definitions_cxx.hpp

```diff
@@ -990,6 +990,14 @@ enum GridDefaultActiveMode : uint8_t {
 	GridDefaultActiveModeMaxElement // Keep as boundary
 };
 
+enum SliceModeDefault : uint8_t  {
```

Comment:
this is the same as SampleRepeatMode so you don't need it

Reply:
deleted it